libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/member/ExportedRoomMemberMapper.kt

```diff
@@ -21,10 +21,10 @@ import io.element.android.libraries.matrix.api.room.RoomMember
 import io.element.android.libraries.matrix.api.room.RoomMembershipState
 import uniffi.matrix_sdk.RoomMemberRole
 import org.matrix.rustcomponents.sdk.MembershipState as RustMembershipState
-import org.matrix.rustcomponents.sdk.ExportedRoomMember as RustExportedRoomMember
+import org.matrix.rustcomponents.sdk.RoomMember as RustRoomMember
 
 object ExportedRoomMemberMapper {
```

Comment:
Should we rename this struct and the file again too?

Reply:
Done!